src/librepgp/packet-print.c

```diff
@@ -1023,143 +1005,169 @@ pgp_sprint_pubkey(const pgp_key_t *key, char *out, size_t outsize)
     return cc;
 }
 
-/**
-\ingroup Core_Print
-\param type
-\param seckey
-*/
 static void
-print_seckey_verbose(const pgp_content_enum type, const pgp_seckey_t *seckey)
+print_seckey_verbose(pgp_printstate_t *     printstate,
+                     const pgp_content_enum type,
+                     const pgp_seckey_t *   seckey)
 {
-    printf("------- SECRET KEY ------\n");
-    print_tagname(0, (type == PGP_PTAG_CT_SECRET_KEY) ? "SECRET_KEY" : "SECRET_SUBKEY");
     /* pgp_print_pubkey(key); */
```

Comment:
I think it makes sense to print the pubkey here too. I added `pgp_print_pubkey(printstate->indent, &seckey->pubkey);` locally and it produces some decent output (maybe a bit messy).

Reply:
Good point.